pcsx2-qt/Settings/SettingsDialog.cpp

```diff
@@ -56,6 +56,7 @@ SettingsDialog::SettingsDialog(QWidget* parent)
 	, m_game_crc(0)
 {
 	setupUi(nullptr);
+	QTHOST_WIDGET_MOUSE_WHEEL_FILTER()
```

Comment:
Apparently you can install multiple filters per widget, this doesn't conflict with `registerWidgetHelp`

Reply:
Not really sure why you need the mess of a macro here. You could just create the filter object in a method inside `SettingsDialog` (parenting it to the dialog), and iterate the widgets there.

Reply:
> Not really sure why you need the mess of a macro here. You could just create the filter object in a method inside `SettingsDialog` (parenting it to the dialog), and iterate the widgets there.

Other than the idea slipped my mind, nothing :laughing: